eq-author/src/components/ContentPickerModal/index.test.js

```diff
@@ -138,76 +136,11 @@ describe("ContentPickerModal", () => {
       <ContentPickerModal {...omit(props, "metadataData")} />
     );
     const tabs = wrapper.find(BaseTabs).prop("tabs");
-    const MetadataTab = tabs[2].render;
+    const MetadataTab = tabs[1].render;
     const metadataTabWrapper = shallow(<MetadataTab />);
     expect(metadataTabWrapper).toMatchSnapshot();
   });
 
-  it("should submit with a question when the user picks a question", () => {
-    const wrapper = shallow(<ContentPickerModal {...props} />);
-    const tabs = wrapper.find(BaseTabs).prop("tabs");
-    const QuestionTab = tabs[1].render;
-    const questionTabWrapper = shallow(<QuestionTab />);
-    questionTabWrapper
-      .find(QuestionContentPicker)
-      .simulate("submit", { id: "2", displayName: "Question" });
-    expect(props.onSubmit).toHaveBeenCalledWith({
-      id: "2",
-      displayName: "Question",
-    });
-  });
-
-  it("should show an error message when there is no question", () => {
-    const wrapper = shallow(
-      <ContentPickerModal {...props} questionData={[]} />
-    );
-    const tabs = wrapper.find(BaseTabs).prop("tabs");
-    const QuestionTab = tabs[1].render;
-    const questionTabWrapper = shallow(<QuestionTab />);
-    expect(questionTabWrapper).toMatchSnapshot();
-  });
-
```

Comment:
I'm assuming these removed tests from this file are now covered in the other tests file like RoutingDestinationContentPicker.test.js or the functionality has been removed in this PR?

Reply:
Yes, this functionality has been removed. These tests were testing the old content picker. If you look in `src/components/ContentPickerModal/index.js`, I removed lots of stuff there because it wasn't being used. So these tests are not relevant anymore.

Reply:
coolio!